azext_iot/product/_help.py

```diff
@@ -0,0 +1,173 @@
+            az iot product requirement list
+    """
+# product tests
+    helps["iot product test"] = """
+        type: group
+        short-summary: Manage device tests for product certification
+    """
+    helps["iot product test create"] = """
+        type: command
+        short-summary: Create a new product test for product certification
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test create --configuration-file {configuration_file}
+        - name: Generate provisioning configuration info from the server
+          text: >
+            az iot product test create --configuration-file {configuration_file} --provisioning
+    """
+    helps["iot product test init"] = """
+        type: command
+        short-summary: Used to initialize local workspace for a new product certification
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test init --product-name {product_name}
+        - name: Specify working folder
+          text: >
+            az iot product test init --product-name {product_name} --working-folder {working_folder}
+    """
+    helps["iot product test search"] = """
+        type: command
+        short-summary: Search product repository for testing data
+        examples:
+        - name: Search by product id
+          text: >
+            az iot product test search --product-id {product_id}
+        - name: Search by DPS registration
+          text: >
+            az iot product test search --registration-id {registration_id}
+        - name: Search by x509 certifcate common name (CN)
+          text: >
+            az iot product test search --certificate-name {certificate_name}
+        - name: Search by multiple values
+          text: >
+            az iot product test search --product-id {product_id} --certificate-name {certificate_name}
+    """
+    helps["iot product test show"] = """
+        type: command
+        short-summary: View product test data
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test show --test-id {test_id}
+    """
+    helps["iot product test update"] = """
+        type: command
+        short-summary: Update the product certification test data
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test update --test-id {test_id} --configuration-file {configuration_file}
+    """
+# Product Test Tasks
+    helps["iot product test task"] = """
+        type: group
+        short-summary: Manage product testing certification tasks
+    """
+    helps["iot product test task create"] = """
+        type: command
+        short-summary: Queue a new testing task. Only one testing task can be running at a time
+        examples:
+        - name: Basic usage
+          text: >
+            az iot product test task create --test-id {test_id}
+        - name: Monitor to completion and return test case
```

Comment:
Is monitor the right terminology? We've used 'polling' terminology in the past (iot hub jobs) to indicate if the command should block until a completion state has occurred